doc/recipes.rst

```diff
@@ -333,8 +333,8 @@ Refreshing modified Templates when OPcache or APC is enabled
 ------------------------------------------------------------
 
 When using OPcache with ``opcache.validate_timestamps`` set to ``0`` or APC
-with ``apc.stat`` set to ``0`` and Twig cache enabled, clearing the template
-cache won't update the cache.
+with ``apc.stat`` set to ``0``, Twig cache enabled and auto refresh disabled,
```

Comment:
Maybe it's time to remove APC information here (and in the title)? Sorry, it's not related, but that's maybe the occasion to cleanup this recipe a bit :)

Reply:
Yes, done.